test/conformance/testdata/conformance.yaml

```
@@ -1678,6 +1678,15 @@
     Pod and the service must now have empty set of endpoints.
   release: v1.9
   file: test/e2e/network/service.go
+- testname: Endpoint resource lifecycle
+  codename: '[sig-network] Services should test the lifecycle of an Endpoint [Conformance]'
+  description: Create an endpoint, the endpoint MUST exist. The endpoint is updated
+    with a new label, a check after the update MUST find the changes. The endpoint
+    is then patched with a new IP address and port, a check after the patch MUST the
```

Is
```suggestion
    is then patched with a new IPv4 address and port, a check after the patch MUST the
```
also true? _(please do feel free to merge without addressing this query)_